### ISSUE TITLE
Nested @if, @else and containing rules aren't linted

### ISSUE THREAD

**Issue by dustindowell22** [brigade/scss-lint#504]
I don't get any notification when this happens next to comment 1, but it works fine on comment 2.

This also prevents any rules inside the `@if` or `@else` to be linted, like in comment 3, the `@extend` should be above `color`. 

```scss
$foo: bar;
$color: #00baff;
$color2: #d3d3d3;

%foobar {
  background: $color2;
}

@if $foo == bar {
  @if $foo == bar {
    color: $color;
  }
  @else if $foo == baz { // #1
    color: $color;
  }
} @else if $foo == qux { // #2
  color: $color;
  @extend %foobar; // #3
}
```

**Comment by sds** [comment]
Thanks for the report, @dustindowell22!

Fixed in bcd8ceaa08347332065a73617f73c22aa285d203.
